test/unit/api/profile-getters.test.ts

```diff
@@ -0,0 +1,138 @@
+import { describe, expect, it } from "bun:test";
+import type { Patient } from "../../../examples/typescript-us-core/fhir-types/hl7-fhir-r4-core/Patient";
+import type { Observation } from "../../../examples/typescript-us-core/fhir-types/hl7-fhir-r4-core/Observation";
+import { USCorePatientProfileProfile } from "../../../examples/typescript-us-core/fhir-types/hl7-fhir-us-core/profiles/UscorePatientProfile";
+import { USCoreBloodPressureProfileProfile } from "../../../examples/typescript-us-core/fhir-types/hl7-fhir-us-core/profiles/UscoreBloodPressureProfile";
+
+const createPatient = (): Patient => ({ resourceType: "Patient" });
+const createObservation = (): Observation => ({ resourceType: "Observation", status: "final", code: {} });
+
+describe("Profile Getter Methods", () => {
+	describe("Extension getters", () => {
+		it("returns simplified object when raw=false (default)", () => {
+			const profile = new USCorePatientProfileProfile(createPatient());
+			profile.setRace({
+				ombCategory: { system: "urn:oid:2.16.840.1.113883.6.238", code: "2106-3", display: "White" },
+				text: "White",
+			});
+
+			const result = profile.getRace();
+			expect(result).toBeDefined();
+			expect((result?.ombCategory as { code?: string })?.code).toBe("2106-3");
+			expect(result?.text).toBe("White");
+		});
+
+		it("returns raw Extension when raw=true", () => {
+			const profile = new USCorePatientProfileProfile(createPatient());
+			profile.setRace({
+				ombCategory: { system: "urn:oid:2.16.840.1.113883.6.238", code: "2106-3", display: "White" },
+				text: "White",
+			});
+
+			const raw = profile.getRace(true);
+			expect(raw).toBeDefined();
+			expect(raw?.url).toBe("http://hl7.org/fhir/us/core/StructureDefinition/us-core-race");
```

Comment:
Looks like having a `raw` option for setter may be a good idea. Just for consistency. Or not because it should be a very strange case with `url` assertion inside.